libs/wingsdk/src/target-sim/app.ts

```diff
@@ -71,9 +67,6 @@ export class App extends core.App {
       case BUCKET_FQN:
         return new Bucket(scope, id, args[0]);
 
-      case LOGGER_FQN:
-        return new Logger(scope, id);
-
       case QUEUE_FQN:
```

Comment:
not related to your change, but there are two `case TOPIC_FQN:` here (line 73 and 82)

Reply:
Nice catch